v2.6/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calicoctl.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
   namespace: kube-system
 spec:
   hostNetwork: true
+  serviceAccountName: calico-node
```

Comment:
Don't we also need the ClusterRole, etc in this file? And the one below?

Reply:
We do, as I said in my last comment I just wanna get it fleshed out in master then I'll make the same changes in the other yamls.